types/src/block/output/mod.rs

```diff
@@ -267,11 +267,11 @@ impl Output {
         &self,
         current_time: u32,
         output_id: &OutputId,
-        alias_state_transition: bool,
+        alias_transition: Option<AliasTransition>,
     ) -> Result<(Address, Option<Address>), Error> {
         match self {
             Self::Alias(output) => {
-                if alias_state_transition {
+                if alias_transition.unwrap_or(AliasTransition::State) == AliasTransition::State {
```

Comment:
Are we fine with state transition as default here ?

Reply:
I'm fine with it if you add it to the comment of the function